shuup/admin/modules/media/static_src/media/browser/menus/fileContextMenu.js

```diff
@@ -6,8 +6,7 @@
  * This source code is licensed under the OSL-3.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-const fileActions = require("../actions/fileActions");
```

Comment:
this is missing
![image](https://user-images.githubusercontent.com/8770370/41131978-ac847688-6a94-11e8-9411-b8820e46a3a3.png)